### PR TITLE
Make enumtables schema aware

### PR DESCRIPTION
**Pull request by ttung**
### Description
This library was unaware of schemas and as such choked whenever it came across a schema.

### Test plan
used in later PR.

### CODE REVIEW

File: third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py [hunk 1]
```diff
@@ -36,15 +36,19 @@ def compare_enums(autogen_context, upgrade_ops, schema_names):
...
 		for table, values in enums.items():
-			if is_table_present(table, autogen_context.connection):
-				items = {r[0] for r in autogen_context.connection.execute("SELECT item_id FROM {}".format(table))}
+			fully_qualified_table_name = "%s.%s" % (schema, table) if schema else table
+			if is_table_present(fully_qualified_table_name, autogen_context.connection):
+				items = {
+					r[0]
+					for r in autogen_context.connection.execute(
+						"SELECT item_id FROM {}".format(fully_qualified_table_name)
+					)
```
Comment (phoenixAja) [inline third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:45]: should we use bind parameters for this? 

`sql = sql.text("SELECT item_id FROM :tablename")`
`autogen_context.connection.execute(sql, tablename=fully_qualified_table_name)`

File: third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py [hunk 2]
```diff
@@ -7,53 +7,64 @@
...
+	def fully_qualified_tablename(self):
+		return "%s.%s" % (self.schema, self.tablename) if self.schema else self.tablename
+
 @Operations.implementation_for(EnumInsertOp)
 def insert(operations, operation):
 	if not operation.data:
 		return
 	values = ', '.join("('" + v + "')" for v in operation.data)
-	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.tablename, vl = values)
+	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.fully_qualified_tablename, vl = values)
```
Comment (phoenixAja) [inline third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py]: same comment as above about bind parameters with these insert statements

File: third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py (hunk 1 above)
Comment (ttung) [reply third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:45]: I'll fix this in #59, as this code actually get yoinked there.

Comment (phoenixAja) [reply third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:45]: ok makes sense

File: third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py (hunk 2 above)
Comment (ttung) [reply third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py]: fixed, and ready for review.

**Review by phoenixAja — APPROVED** [review]

**Pull request merged**